gapic/src/main/com/google/gapid/perfetto/views/FrameEventsSummaryPanel.java

```diff
@@ -176,10 +176,16 @@ public void renderSlices(RenderContext ctx, FrameEventsTrack.Data data) {
       double rectStart = state.timeToPx(tStart);
       StyleConstants.Gradient color = getSliceColor(data.titles[i]);
       color.applyBase(ctx);
+      if (!selectedFrames.isEmpty()) {
+        ctx.setBackgroundColor(color.disabled);
+        if (selectedFrames.contains(data.frameNumbers[i], data.layerNames[i])) {
+          color.applyBase(ctx);
+        }
+      }
 
       if (tEnd - tStart > 3 ) {
         double rectWidth = Math.max(1, state.timeToPx(tEnd) - rectStart);
-        double y = depth * SLICE_HEIGHT;
+        double y = 0;
```

Comment:
Removed the depth from the query altogether because its just not being used anywhere. And currently, there is no plan of using it in the foreseeable future as well.  Even if we change the instant events to slices, the depth will always be 0. We can always add it back in case it is needed, but its just a waste for now, with so many select queries being sent.